### DISCUSSION TITLE
Translate source string and translation string at the same time

### DISCUSSION THREAD

**Discussion by Roelsoft** [WeblateOrg/weblate#8777] [Q&A]
Hello,

Is it possible to configure Weblate so that you can translate both the source string (English in our case) and the translation at the same time? (In this case Dutch). 

Right now - I have to switch between English and Dutch for the same key to translate both. One of the two is always read-only. 

Example:
![image](https://user-images.githubusercontent.com/107626768/218051016-4bf5cc8d-9a0f-4618-8e9e-604b8559c3df.png)

Is there a way so we can edit both English and Dutch at the same time?

**Comment by nijel** [comment]
Not right now, see https://github.com/WeblateOrg/weblate/issues/197
